### ISSUE TITLE
Vanilla installer does not set /etc/vconsole.conf

### ISSUE THREAD

**Issue by VladShmi** [Vanilla-OS/Albius#40]
I am experiencing an issue during the boot process of VanillaOS Orchid where I am unable to select the keyboard layout to decrypt the disk with LUKS2. It's by default in US layout and you can't change it anywhere. That makes it difficult  to enter the correct credentials to decrypt disk every time at boot.

**Comment by axtloss** [comment]
> @ muhdsalm It might be, but it's not clear yet.
> 
> Just setting vconsole.conf sadly doesn't fix the issue.

Yes it does. As I already stated in a comment previously, the initramfs build process sources /etc/vconsole.conf to decide which keyboard layout to use. Albius does not set /etc/vconsole so the keyboard layout gets defaulted to us.

**Comment by taukakao** [comment]
@axtloss I tested it with a custom image and also set vconsole in the installer and it didn't change anything. 

https://github.com/taukakao/ontop-desktop/pkgs/container/tests/161012673?tag=german-locale

source:
https://github.com/taukakao/ontop-desktop/compare/main...german-locale

**Comment by axtloss** [comment]
Your vconsole configuration is wrong.

**Comment by Thatoo** [comment]
On a # 113 beta version installed in a VM, I added `KEYMAP="fr-oss"` to `/etc/vconsole.conf` with
`host-shell pkexec abroot nano /etc/vconsole.conf`
and I update intramfs with
`host-shell pkexec abroot update-initramfs`
it didn't work.
Then I tried changing to `KEYMAP=fr` only and changed in `/etc/initramfs-tools/initramfs.conf`
```
#
# KEYMAP: [ y | n ]
#
# Charger une configuration de clavier à l'étape d'initramfs.
#

KEYMAP=y
```
and I updated again intramfs with
`host-shell pkexec abroot update-initramfs`
but still it doesn't work.

Fedora silverblue face a similar issue and don't manage to solve it at install but at least there is a guide to solve the issue after install https://docs.fedoraproject.org/en-US/fedora-silverblue/installation/#known-limitations
The command is
`rpm-ostree initramfs-etc --track=/etc/vconsole.conf`

I hope we can soon at least have a similar guide to solve this issue post-installation if it's difficult to solve the issue for now at installation time.

**Comment by taukakao** [comment]
axtloss said it has something to do with missing keymaps or something, which apparently is a debian issue but I haven't looked into it.

**Comment by taukakao** [comment]
So it seems I accidentally fixed this problem in ABRoot with https://github.com/Vanilla-OS/ABRoot/commit/d2508d189414c754d82c32b2325e450e85431515

This commit just added the /etc/ overlay to the chroot so the mkinitramfs would see /etc/default/keyboard (/etc/vconsole.conf) 

Not sure if this is the only relevant config here but I guess something similar could be done for the installer.

**Comment by taukakao** [comment]
This can prevent the user from decrypting their device, so I'm adding this back to the stable milestone.

**Comment by taukakao** [comment]
I took a quick look and it's because initramfs generation is part of the install process 
https://github.com/Vanilla-OS/Albius/blob/9fb81a0943f1d79cefbfd097009dc6afa967e639/core/recipe.go#L1122
while setting the language is part of the post installation steps. 
https://github.com/Vanilla-OS/Albius/blob/9fb81a0943f1d79cefbfd097009dc6afa967e639/core/recipe.go#L769

While it makes the code a bit more complex I think we probably have to run the Initramfs generation after the post install scripts here:
https://github.com/Vanilla-OS/Albius/blob/9fb81a0943f1d79cefbfd097009dc6afa967e639/core/recipe.go#L914

I will get on this as soon as I can, probably Wednesday.
